generic/CMakeLists.txt

```
@@ -56,7 +56,7 @@ find_package(GFTL_SHARED REQUIRED)
 esma_add_library(${this}
   SRCS ${srcs}
   DEPENDENCIES MAPL.oomph MAPL.shared MAPL.profiler MAPL.base
-               PFLOGGER::pflogger GFTL_SHARED::gftl-shared-v2 GFTL_SHARED::gftl-shared-v1 GFTL::gftl
+               PFLOGGER::pflogger GFTL_SHARED::gftl-shared-v2 GFTL_SHARED::gftl-shared-v1 GFTL::gftl GFTL::gftl-v2 GFTL::gftl-v1
```

We want to be thorough, but not redundant.
```suggestion
               PFLOGGER::pflogger GFTL_SHARED::gftl-shared-v2 GFTL_SHARED::gftl-shared-v1  GFTL::gftl-v2 GFTL::gftl-v1
```

---

Your suggestion seems fine to me. Why do you take it back?